contracts/smart-account/libs/AddressResolver.sol

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
```

Comment:
use solidity ^0.8.20

Reply:
once your PR is merged would be better to do this just for this contract.